src/OnboardingSPA/components/Drawer/DrawerPanel/DesignThemeStylesPreview.js

```diff
@@ -36,6 +38,10 @@ const DesignThemeStylesPreview = () => {
 		updateThemeStatus,
 	} = useDispatch( nfdOnboardingStore );
 
+	const THEME_VARIATIONS
+		= window.nfdOnboarding?.themeStepData[currentStep?.patternId]?.styles
+		* window.nfdOnboarding?.themeStepData[currentStep?.patternId]?.patterns;
+		
```

Comment:
Computation to backend refer relevant comment

Reply:
Sure